src/desktop/chat/chatwidget.h

```diff
@@ -79,9 +79,13 @@ private slots:
 
 private:
 #ifdef Q_OS_ANDROID
+	// On Android, image resources don't work, leading to avatars not showing.
```

Comment:
Why don’t they work?

Reply:
Avatars in chat use a special kind of URL and apparently Android has some kind of URL restrictions that make it not work with those. I didn't look too deep into after prior dives into Qt's Android support, I just turned them off instead.